docs/examples/example_saturation.py

```diff
@@ -0,0 +1,79 @@
+from pyrokinetics import Pyro, PyroScan
+from pyrokinetics.diagnostics.saturation_rules import SaturationRules
+import numpy as np
+
+# Choose convention
+output_convention = "pyrokinetics"
+
+# Base file
+base_dir = "/common/CSD3/ir-giac2/t3d/resolution/nth128/restart6/outputs"
```

Comment:
This is probably fine because it provides just an example of usage, but most users wont be able to access to this folder.

Reply:
Yeah I agree. I wonder if it is worth having data somewhere that we can pull and use as an example. This would be useful for basically a bunch of testing as we could have better quality testing.